### DISCUSSION TITLE
Horizontal layout

### DISCUSSION THREAD

**Discussion by spacekomet** [URUWorks/TeroSubtitler#200] [Ideas]
Hello

Currently we have subtitles list on the left side, video preview on the right, separated vertically and sub editor at the bottom.
Isn't it more conveniently if there would be an option to also split list and video horizontally, e.g. video above, list below and editor at the bottom? Specially when you have translate mode on. 

Or you have all bigger wide screens?  :)

**Comment by URUWorks** [comment]
In that case it is possible to detach the video and order both forms as you like :)

**Reply by spacekomet** [reply]
Thanks for the hint :).
